### ISSUE TITLE
[Bug Report] Edits "Updated At"

### ISSUE THREAD

**Issue by SpedNSFW** [stashapp/stash-box#501]
**Describe the bug**
An edit's "updated at" time is not changed when the edit changes status (e.g. from pending to accepted).

**To Reproduce**
1. Make two separate edits
2. Have the most recent one be accepted first, then the first one
3. Filter edits by "accepted" and sort by "updated at". They won't appear in the correct order.

**Expected behavior**
"Updated at" time should be updated when an edit changes status, so that looking at edits, filtered by accepted and sorted by updated at should show the most recently updated at the top (unless a comment is made)

**Comment by peolic** [comment]
> 3. Filter edits by "accepted" and sort by "updated at". They won't appear in the correct order.

TLDR:
you should now be sorting by *Closed At* instead. The *Updated At* field was repurposed for the edit amending feature (#365).

---

the `updated_at` field was repurposed in #365 *"to denote if and when an edit was amended"*.

> so that looking at edits, filtered by accepted and sorted by updated at should show the most recently updated at the top (unless a comment is made)

ever since the PR linked above, the edit ordering needed to be fixed.
with #478 being merged, it should now be fixed.

sorting by `updated_at` or `closed_at` will now fall back to `created_at`, making the edits appear in the correct order.

---

I'm not sure, but if you are asking for an edit order option that combines the three (making the order `closed_at`, `updated_at`, `created_at`), then this should be a feature request now, in my opinion.
